content/lessons/testing/testing-workshop.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Testing Workshop
+template: lesson
+draft: false
+slug: /courses/Testing/testing-workshop
+course: Testing
+tags:
+  - Jest
+description: "If you've made to this section, you likely already have a good grasp on frontend testing. This workshop will challenge you to hone your skills with an open ended approach. The workshop features a simple app implemented in a few different ways. You job is to provide tests for each app. We've provided 'tickets' with testing directions to keep you on track. Check out the video below for more information"
+timeToCompletion: ~2-4 hours
+secondaryExerciseUrl: 'https://codesandbox.io/s/github/Cooperbuilt/frontend-testing-workshop'
```

Comment:
Make sure this points to `wayfair/awesome-learning-exercises`

Reply:
I am not ready to add the workshop to the exercises repo. This is why I extended the button to allow for outside links. I have a plan to move it to the exercises repo, but I don't want it to be a blocker for this PR.